runtime/runtime.go

```diff
@@ -320,7 +320,8 @@ func (r *interpreterRuntime) interpret(
 		exportedValue = newExportableValue(result, inter)
 	}
 
-	return exportedValue, inter, nil
+	err = inter.ExitHandler()
```

Comment:
CI is failing because this handler is currently always called, but it is optional:
```suggestion
	if inter.ExitHandler != nil {
		err = inter.ExitHandler()
	}
```